Jenkins/Pipelines/Terraform/Jenkinsfile

```diff
@@ -0,0 +1,56 @@
+#!groovy
+
+pipeline {
+  agent any
+
+  stages {
+    stage('FetchCode'){
+      steps {
+        git url: "https://github.com/uc-cdis/cloud-automation.git", branch: 'fix/variable-state'
```

Comment:
the branch should be just whatever branch of active prs that jenkins found?

Reply:
This pipeline is configured to run by hand.  
Once we merge can switch to 'master' branch.
The idea is we could run terraform through a jenkins job rather than by hand, but that may not make sense.
The TerraformPlanOnly/JenkinsFile is configured to run with a 'MultiBranch' setup - it has a
    scm checkout
line ...

Reply:
I see, so this is the first clone that will only be run once when we setup jenkins?
let's switch to master before we merge?

Reply:
we point Jenkins at which Jenkinsfile we want it to run when we configure a 'job', and there are different kinds of jobs with different rules.  Let's do a walk-through of what I have setup next week.  It's just v1 - definitely need to iterate :-)

Reply:
yep!